cmd/containerboot/main.go

```diff
@@ -772,6 +829,26 @@ func ensureIPForwarding(root, clusterProxyTarget, tailnetTargetiP, tailnetTarget
 			v6Forwarding = true
 		}
 	}
+	if clusterProxyTargetDNSName != "" {
```

Comment:
Consider using `else if` since `clusterProxyTargetIP` and `clusterProxyTargetDNSName` are mutually exclusive.

Reply:
We would fail at [validation if both were specified](https://github.com/tailscale/tailscale/pull/11802/commits/477b8d0483b6caec5a8998791f92b363e7ace1ab#diff-700830ac79a41da42cd3a82065f0fee3003a4390eead4db1de4bf30557a06bd6R1116)

Reply:
Yes, I saw the validation, just thought it would be nice to reinforce that these are mutually exclusive in this branching logic. Not a big deal either way.

Reply:
I've left this as is for now - I would find the code blocks harder to read myself with even more indentation. We do need to refactor the containerboot code as it is growing out quite wildly now, there is an idea to implement a TCP proxy in code

Reply:
Totally cool. FWIW, using `else if` wouldn't add any indentation :)

Reply:
more characters :) `if` -> `else if`